scripts/globals/items/dusty_elixir.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------------
+-- ID: 5433
+-- Item: Dusty Elixir
+-- Item Effect: Instantly restores 25% of HP and MP
+-----------------------------------------
+require("scripts/globals/msg")
+
+function onItemCheck(target)
+    local result = 0
+    local mHP = target:getMaxHP()
+    local cHP = target:getHP()
+    local mMP = target:getMaxMP()
+    local cMP = target:getMP()
+
+    if (mHP == cHP and mMP == cMP) then
```

Comment:
https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#no-parentheses-unless-needed-to-clarify-order-of-operations